content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -120,8 +146,35 @@ def vmdb_object(model, id)
 end
 
 def requested_storage(args_hash)
-  vm_size = args_hash[:resource].vm_template.provisioned_storage
-  args_hash[:prov_value] = args_hash[:number_of_vms] * vm_size
+  if @reconfigure_request
+    args_hash[:prov_value] = 0
+    # Adding/removing disks only supported for VMware
+    if args_hash[:resource].options[:disk_add]
+      args_hash[:resource].options[:disk_add].each do |disk|
+        $evm.log(:info, "Adding a disk:  #{disk.inspect}")
+        args_hash[:prov_value] += disk['disk_size_in_mb'].to_i.megabytes
+      end
+    end
+    if args_hash[:resource].options[:disk_remove]
+      args_hash[:resource].options[:disk_remove].each do |disk|
+        $evm.log(:info, "Removing disk: #{disk.inspect}")
```

Comment:
@billfitzgerald0120 
Can we move this line to line 163 and change the message to  Disk Info